npm-shrinkwrap.json

```diff
@@ -3,5754 +3,21 @@
   "version": "3.12.0",
   "lockfileVersion": 2,
   "requires": true,
-  "packages": {
```

Comment:
This seems like a huge change. Is it intentional?

Reply:
I just updated the packages so not sure above, it came in automatically. Manually I only reverted the lockfileversion from 1 to 2. 
Yes it builds and runs test 100% in my windows machine.  Did not try on my mac.

![image](https://user-images.githubusercontent.com/20379311/124671431-cac38900-def8-11eb-925c-55bcfc5e21b7.png)

Reply:
Hmm, so that could be that the contents of the lockfile are v2 but the version on top is v1 🤔

Reply:
version on top? Pls could you clarify?

Reply:
On line 4 there is the `lockfileVersion` attribute which now says 2, but I wonder if the rest of the contents, including the massive sections that's been removed isn't v1.

Reply:
What version of npm do we use to make sure the lockfileversion is the same when anyone updates a dependency?

Reply:
I use npm@7. In CI/CD we use npm@6. The lockfile is generated using npm@7. That way we verify that it works both with v6 and v7. Lockfile should be v2 indeed, but I'm concerned about the big diff that seems like something more has happened than just updating the dependencies.

Reply:
No worries, I will close this pull request for now, and update npm and check again @waldekmastykarz